tests/end_to_end/test_vulnerability_detector/test_vulnerability_detector.py

```diff
@@ -928,8 +943,10 @@ def test_vulnerability_detector_scans_cases(self, request, preconditions, body,
             package_data = [body["package"]["to"], body["package"]["from"]]
         else:
             package_data = [body["package"]]
+        max_result_window(host_manager)
```

Comment:
Most of the calls to `max_result_window` are done prior to calling the function `get_vulnerabilities_index`

Reply:
Check https://github.com/wazuh/wazuh-qa/pull/5712#discussion_r1745290989. In addition I prefer explicitly include this operation in the test rather than include not expected logic in `get_vulnerabilities_index`